backend/internal/gateway/coderabbit_feedback_test.go

```
@@ -0,0 +1,96 @@
+	}
+}
+
+func TestResolveInstanceStorageFailureReturnsGenericResponse(t *testing.T) {
+	fixture := newGatewayFixture(t, false)
+	if err := fixture.db.Close(); err != nil {
+		t.Fatal(err)
+	}
+
+	recorder := httptest.NewRecorder()
+	observed := newResponseObserver(recorder, false)
+	request := httptest.NewRequest(http.MethodGet, "/v1/models/gateway-model", nil)
```

golangci-lint (2.13.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)